### PR TITLE
Update IntuosV2 parser to match input-wacom

### PR DESCRIPTION
**Pull request by X9VoiD**
X and Y, [wacom_wac.c#L884](https://github.com/linuxwacom/input-wacom/blob/master/4.5/wacom_wac.c#L884)
Pressure, [wacom_wac.c#L904](https://github.com/linuxwacom/input-wacom/blob/master/4.5/wacom_wac.c#L904)

- [x] Testing (by AbstractQBit)

### CODE REVIEW

**Comment by AbstractQbit** [comment]
Pressure now works correctly (up to 2047 instead of 2040), but the cursor is flying all over the screen. BitConverter probably converts with the wrong endiannes.

**Comment by AbstractQbit** [comment]
I'd suggest leaving it as 
```
var x = ((report[2] * 0x100) + report[3]) << 1 | ((report[9] >> 1) & 1);
var y = ((report[4] * 0x100) + report[5]) << 1 | (report[9] & 1);
```

**Comment by X9VoiD** [comment]
If it still doesn't work then I guess it's wacom magic, and we go with your suggestion.

**Review by InfinityGhost — APPROVED** [review]

File: OpenTabletDriver/Vendors/Wacom/IntuosV2Report.cs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+using System;
```
Comment (InfinityGhost) [inline OpenTabletDriver/Vendors/Wacom/IntuosV2Report.cs:1]: Is this needed?

Comment (X9VoiD) [reply OpenTabletDriver/Vendors/Wacom/IntuosV2Report.cs:1]: Oh, uhhh... not needed anymore lmao

Comment (X9VoiD) [reply OpenTabletDriver/Vendors/Wacom/IntuosV2Report.cs]: Change pushed

**Pull request merged**